freertos_gdb/queue.py

```diff
@@ -149,7 +150,7 @@ def get_table_rows(self, q_id):
 def get_task_id_name_str(task):
     task_id = TaskProperty.ID.get_val_as_is(task)
     task_name = TaskProperty.NAME.get_string_val(task)
-    return f'{task_id} {task_name}'
+    return '{task_id} {task_name}'.format(task_id=task_id, task_name=task_name)
```

Comment:
```suggestion
    return '{} {}'.format(task_id, task_name)
```

To have better readability

Reply:
Sure.